fgspectra/cross.py

```diff
@@ -192,7 +195,11 @@ def eval(self, sed_kwargs={}, cl_kwargs={}):
         """
 
         f_nu = self._sed(**sed_kwargs)
-        return np.einsum('k...i,n...j,...knl->...ijl',
+        if np.shape(f_nu[:,0]) != (2,):
```

Comment:
This branching condition (as well as the one above) breaks the generality of the ellipsis. We have to find a better solution @umbnat92

Reply:
here I'm not getting what you mean

Reply:
The `...` in einsum are meant to allow for extra dimensions. However, the `if` you propose does not allow that.

Reply:
Oh, ok. Now I see. Thanks! [Here](https://github.com/simonsobs/fgspectra/blob/52359184ed3fbba1f4ab0895ab9bc970ba16cf48/fgspectra/cross.py#L142) it is ok, right? Because I am checking that the result of the bandpass integration does not have an ell-dependency.